dts/bindings/wifi/nordic,nrf700x-qspi.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+
+description: This is a representation of the nRF700x Wi-Fi chip.
+
+compatible: "nordic,nrf700x-qspi"
+
+include: "nordic,nrf700x.yaml"
+
+on-bus: qspi
+
+properties:
+  sck-frequency:
+    type: int
+    required: true
+    description: |
+      Maximum clock speed (in Hz) supported by the device.
+
+  reg:
+    required: true
+
+  quad-mode:
+    type: boolean
+    description: |
+      If specified, Use QSPI in quad mode (4 IO lines) otherwise in
+      SPI mode (2 IO lines - MOSI & MISO).
+
+  rx-delay:
+    type: int
+    description: |
+      Number of clock cycles from the rising edge of the SPI clock
+      until the input serial data is sampled.
+
+  cpha:
+    type: boolean
+    description: |
+      Set to indicate phase starts with asserted half-phase (CPHA=1).
+      The driver using this property must also use `cpol`.
+
+  cpol:
+    type: boolean
+    description: |
+      Set to indicate that the clock leading edge is falling (CPOL=1).
+      The driver using this property requires also use `cpha`.
```

Comment:
The `spi-device` base binding uses the names/descriptions `spi-max-frequency`, `spi-cpha` and `spi-cpha`, could we use the same names here for consistency?

Reply:
Ah sure, as they were under the (Q)SPI heading dropped the prefix, but I can reinstate that.